### PR TITLE
Add Pipelines for keyvault/internal and prepare for release

### PR DESCRIPTION
**Pull request by seankane-msft**
Creates pipelines for keyvault/internal module and prepares for release. Also adds the constants.go file with a version, a LICENSE.txt, ci.keyvault.yml file, and an eng/config.json entry for code coverage

### CODE REVIEW

File: sdk/keyvault/internal/constants.go [hunk 1]
```diff
@@ -0,0 +1,11 @@
+//go:build go1.16
+// +build go1.16
+
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+package internal
+
+const (
+	version = "0.1.0" //nolint
```
Comment (weshaggard) [inline sdk/keyvault/internal/constants.go:10]:
```suggestion
	version = "v0.1.0" //nolint
```

**Review by richardpark-msft — APPROVED** [review]
Looks good, especially for an internal package.

**Pull request merged**